### DISCUSSION TITLE
How do I set the event types  5 , special sampling "-" event color？

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1120] [Q&A]
![20230305101858](https://user-images.githubusercontent.com/116416332/222938142-78ef23be-0209-40f4-94f6-adb28eb4d460.png)
![20230305101952](https://user-images.githubusercontent.com/116416332/222938144-050408f0-efb9-4323-9951-306d938e3ebe.png)

**Comment by roasterdave** [comment]
Config>> Colors>> Graph tab>> Special Event Marker and Special Event Text

![image](https://user-images.githubusercontent.com/30009312/222946573-7ef4d23d-f2be-4bf7-a6a9-e5e6b7a2d40c.png)
